README.rst

```diff
@@ -19,6 +19,7 @@ Key Features
 - Modern Pythonic API using ``async`` and ``await``.
 - Proper rate limit handling.
 - Optimised in both speed and memory.
+- Supports Slash Commands, Context Menus and Message Components.
```

Comment:
I think we should have something like "Supports Slash Commands, Context Menus and Message Components via gateway". I am not sure though if we do slash commands through http or the gateway.

Reply:
All events are received through the gateway, altough commands are registered through http requests.